src/main/java/emu/grasscutter/command/Command.java

```diff
@@ -9,8 +9,6 @@
 
     String usage() default "No usage specified";
 
-    String description() default "No description specified";
```

Comment:
Instead of removing this you can make the description take the translation key instead

Reply:
OK, I'll change it back

Reply:
I added it back